README.md

```diff
@@ -4,6 +4,9 @@
 
 Package gearman provides a thread-safe Gearman client
 
+### Development
+
+All imports are written with the `gopkg.in` paths so you will have a much easier time editting this package in your `$GOPATH/src/gopkg.in/gearman.v2` directory.
```

Comment:
This repo uses `godocdown`, so technically this file is autogenerated and you shouldn't edit it.

I'm fine with this edit, though, since I'd like get rid of godocdown in this repo and instead

- link to the public docs generated on godoc.org
- explain how to publish to gopkg.in